Resources/views/library/claroline/layout.html.twig

```diff
@@ -166,13 +166,13 @@
         </div>
     </div>
     <div class="container-fluid">
-        <div class="row">
+        {#<div class="row">
             <div class="col-md-12">
                 <ul class="breadcrumb">
                     {% block breadcrumb %}{% endblock %}
                 </ul>
             </div>
-        </div>
+        </div>#}
```

Comment:
AFAIK the breadcrumb block is still part of the global layout. Why should it be commented/removed?